reports/post_diagnostics.Rmd

```diff
@@ -124,36 +126,27 @@ kable(log_status %>%
 We expect all cumulative variables to increase with each subsequent scraper run. Rows highlighted below indicate that the aggregated total for a given variable fell in the latest scrape. This is based on totals from our scraped data in `adult_facility_covid_counts.csv`. 
 
 ``` {r check_previous, echo = F, message = F, warning = F}
-# Read old facility data 
-old_df <- stringr::str_c(
+# Read old national aggregate counts 
+old_total <- stringr::str_c(
     "https://raw.githubusercontent.com/uclalawcovid19behindbars/data/master/", 
-    "latest-data/latest_facility_counts.csv") %>% 
-    read_csv()
+    "latest-data/latest_national_counts.csv") %>% 
+  read_csv() %>%
+  rename(`Old Total` = Count)
 
 covid_suffixes <- c(
   ".Confirmed", ".Deaths", ".Tadmin", ".Active", ".Initiated", ".Completed")
 
 # Get totals 
 rowAny <- function(x) rowSums(x) > 0
 
-new_total <- new_df %>% 
-    filter(rowAny(across(ends_with(covid_suffixes), ~ !is.na(.x)))) %>%
-    select(all_of(ends_with(covid_suffixes))) %>%  
-    summarise_all(sum_na_rm) %>% 
-    pivot_longer(cols = everything(), 
-                 names_to = "Variable", 
-                 values_to = "New Total") 
-
-old_total <- old_df %>% 
-    select(all_of(ends_with(covid_suffixes))) %>% 
-    summarise_all(sum_na_rm) %>% 
-    pivot_longer(cols = everything(), 
-                 names_to = "Variable", 
-                 values_to = "Old Total")
+new_total <- read_csv("../data/latest-data/latest_national_counts.csv") %>%
+  rename(`New Total` = Count)
 
-joined <- full_join(old_total, new_total, by = "Variable") %>% 
-    mutate(Difference = `New Total` - `Old Total`) %>% 
-    mutate(cumulative_flag = ifelse(str_detect(Variable, ".Active|.Initiated|.Completed"), 0, 1))
+joined <- full_join(old_total, new_total, by = "Measure") %>% 
```

Comment:
In this function would it also be possible to have the old and new date. I know Josh and Amanda have asked for the difference since when before and if we can get it in the data directly maybe we can head off a few questions by just adding it as a column.

Reply:
I love this suggestion but unfortunately our ntl counts doesn't have a date column associated...not sure how to quickly and easily get at this beyond communicating that the difference is between the most recent scrape and the previous one